### PR TITLE
Add timezone configuration

### PR DESCRIPTION
**Pull request by yhjhoo**
## What I did

1. Add timezone configuration sample

### CODE REVIEW

File: docs/docs/test-runner/browser-launchers/overview.md [hunk 1]
```diff
@@ -46,3 +46,19 @@ export default {
   browsers: [chromeLauncher({ launchOptions: { args: ['--no-sandbox'] } })],
 };
 ```
+
+
+### Configuring browser timezone
+```js 
+import { playwrightLauncher } from '@web/test-runner-playwright';
```
Comment (web-padawan) [inline docs/docs/test-runner/browser-launchers/overview.md:53]: Please consider moving this code snippet to Playwright launcher documentation page.

**Review by web-padawan — CHANGES_REQUESTED** [review]

File: docs/docs/test-runner/browser-launchers/overview.md (hunk 1 above)
Comment (yhjhoo) [reply docs/docs/test-runner/browser-launchers/overview.md:53]: right, I was assumed the configuration is applicable to all browser luncher

Comment (web-padawan) [reply docs/docs/test-runner/browser-launchers/overview.md:53]: It's probably a good idea to change `Configuring browser timezone` to `Providing browser context`.
Also, please provide some text description and not only code snippet.

**Comment by web-padawan** [comment]
Closing in favor of #1793

**Pull request closed without merging**